tests/boot/grub_test_snapshot.pm

```diff
@@ -14,6 +14,11 @@ use testapi;
 
 sub run() {
     my $self = shift;
+
+    assert_screen "inst-bootmenu", 30;
+    sleep 2;
```

Comment:
Why do you need to do a sleep if you've just waited 30 seconds to assert the screen is there?